### DISCUSSION TITLE
How to combine named groups with data attributes

### DISCUSSION THREAD

**Discussion by abdul-alhasany** [tailwindlabs/tailwindcss#11768] [Help] [answered]
Tailwind 3.2 enabled [conditionally styling based on group data attribute](https://tailwindcss.com/blog/tailwindcss-v3-2#data-attribute-variants). Like this:
```html
<div data-size="medium" class="group">
  <div class="group-data-[size=large]:p-8">
    <!-- Will not apply `p-8` -->
  </div>
</div>
```

but it does not seem to support [named groups](https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-nested-groups). For example:
```html
<div data-size="medium" class="group/root">
  <div class="group-data-[size=large]:p-8">
    <!-- Will not apply `p-8` -->
  </div>
</div>
```

I am not sure how to add `group/root` to child selector. I tried `group/root/data-[]`, `group-data/root-[]` and `group-data/root[]` but nothing seems to work.

Is it this feature supported?

**Comment by andymccray** [comment, marked as answer]
This should do the trick for you:

```html
<div data-size="medium" class="group/root">
  <div class="group-data-[size=large]/root:p-8">
    <!-- Will not apply `p-8` -->
  </div>
</div>
```

**Reply by abdul-alhasany** [reply]
thanks!
